examples/gqlgen-federation/gateway/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "gateway-grafbase",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@grafbase/sdk": "~0.20.0"
```

Comment:
add grafbase as a dependency here and start/dev scripts?

Reply:
another idea is to use the grafbase federation gateway and pass the federated schema SDL via path?

Reply:
Yeah, could do.  Though I was just using `npx grafbase@latest` in `start.sh`, which seemed less annoying (having to update versions etc.)

Reply:
> another idea is to use the grafbase federation gateway and pass the federated schema SDL via path?

Maybe, but then I'd have to hardcode/script/instruct people how to build the federated schema SDL.  I can do that if you think we should though

Reply:
True, better to keep the dev server then.